extensions/sql-migration/src/constants/strings.ts

```diff
@@ -82,6 +82,46 @@ export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.a
 
+// Assessment results
+export const ASSESSMENT_RESULTS_PAGE_TITLE = localize('sql.migration.assessment.results.title', "Assessment results");
+export const ASSESSMENT_RESULTS_SUMMARY_LABEL_CAPS = localize('sql.migration.assessment.results.summary.label.caps', "ASSESSMENT RESULTS");
+export const ASSESSMENT_RESULTS_PAGE_HEADER = localize('sql.migration.assessment.results.header', "View assessment results and select database(s) for migration");
+export const DATABASES_ASSESSED_LABEL = localize('sql.migration.database.assessed.label', "Database(s) assessed");
+export const MIGRATION_TIME_LABEL = localize('sql.migration.migration.time.label', "Ready for migration");
+export const ASSESSMENT_FINDINGS_LABEL = localize('sql.migration.assessment.findings.label', "Assessment findings");
+export const MIGRATION_READINESS_LABEL = localize('sql.migration.migration.readiness.label', "Migration readiness");
+export const SUMMARY_TITLE = localize('sql.migration.summary.title', "Summary");
+export const DETAILS_TITLE = localize('sql.migration.details.title', "Details");
+export const ASSESSMENT_SUMMARY_TITLE = localize('sql.migration.assessment.summary.title', "Assessment summary");
+export const READINESS_SECTION_TITLE = localize('sql.migration.readiness.section.title', "Migration readiness of assessed databases in the Server instance");
+export const TOTAL_FINDINGS_LABEL = localize('sql.migration.total.findings.label', "Total findings");
+export const ISSUES_LABEL = localize('sql.migration.issues.label', "Blocking issues");
+export const INSTANCE_FINDING_SUMMARY = localize('sql.migration.instance.finding.summary', "Server instance assessment findings summary");
+export const SEVERITY_FINDINGS_LABEL = localize('sql.migration.severity.findings.label', "Findings by severity");
+export const ASSESSED_DBS_LABEL = localize('sql.migration.assessed.dbs.label', "Assessed databases");
+export const NOT_READY = localize('sql.migration.not.ready', "Not ready");
+export const READY = localize('sql.migration.ready', "Ready");
+export const NEEDS_REVIEW = localize('sql.migration.needs.review', "Needs review");
+export const READY_WARN = localize('sql.migration.ready.warn', "Ready with warnings");
+export const BLOCKERS = localize('sql.migration.ready', "Blockers");
+export const DATABASE_ISSUES_SUMMARY = localize('sql.migration.database.issues.summary', "Database assessment issues summary");
+export const TOTAL_ISSUES_LABEL = localize('sql.migration.total.issues.label', "Total issues found");
+export const SEVERITY_ISSUES_LABEL = localize('sql.migration.severity.issues.label', "Issues by severity");
+export function DB_READINESS_SECTION_TITLE(dbName: string) {
+	return localize('sql.migration.db.readiness.section.title', "Database {0} migration readiness", dbName);
+}
+export function NON_READINESS_DESCRIPTION(issueCount: number) {
+	return localize('sql.migration.non.readiness.description', "The database is not ready to migrate due to {0} blocking issue.", issueCount);
```

Comment:
since issuecount can be > 1 should the text be changed to "... blocking issue(s)."

Reply:
make sense